homeassistant/config_entries.py

```diff
@@ -1327,10 +1327,10 @@ def async_setup_platforms(
         report(
             (
                 "called async_setup_platforms instead of awaiting"
-                " async_forward_entry_setups; this will fail in version 2022.12"
+                " async_forward_entry_setups; this will fail in version 2023.3"
```

Comment:
That means only one release instead of the usual two.

Reply:
Its been deprecated since 2022.8.0.  I was thinking adding another month was a courtesy.

This was deprecated in [cd03c49fc27](https://github.com/home-assistant/core/pull/73806) 

It was properly labeled as a breaking change and got a blog post so I think its been more than enough time.

Reply:
https://github.com/home-assistant/core/pull/73806 lists quite a lot of integrations which need to do their homework. Is that list kept up to date, or is the list shorter now?

Reply:
All of them have been updated except for the ones that have open PRs that are not yet merged